core/src/components/range/range.tsx

```diff
@@ -555,7 +555,7 @@ export class Range implements ComponentInterface {
       printIonWarning(
         `Using ion-range with an ion-label has been deprecated. To migrate, remove the ion-label and pass your label directly into ion-toggle instead.
```

Comment:
The original warning incorrectly refers to `ion-toggle` instead of `ion-range`. 

Should we update this copy to reflect using a slot (outside of just the code snippet)?

e.g.:

> Using ion-range with an ion-label has been deprecated. To migrate, remove the ion-label and pass your label into an element using slot="label" instead.